scripts/scraper-ng/rules/html-warn-macros.js

```diff
@@ -0,0 +1,20 @@
+const rule = require("unified-lint-rule");
+const visit = require("unist-util-visit-parents");
+
+module.exports = rule("html-warn-on-macros", warnOnMacros);
+
+/**
+ * Issue a warning for each macro
+ */
+function warnOnMacros(tree, file) {
```

Comment:
This warns for the compat macro which we require in the other rule. Should we have a list of required / allow listed macros somewhere that both rules could use?

Reply:
Done. Also a nice demonstration of adding preferences to rules. 👍